arshal_time.go

```diff
@@ -214,8 +213,6 @@ func (a *durationArshaler) initFormat(format string) (ok bool) {
 		a.base = 1e3
 	case "nano":
 		a.base = 1e0
-	case "base60": // see https://en.wikipedia.org/wiki/Sexagesimal#Modern_usage
```

Comment:
What a lost opportunity to call this `format:sexagesimal` 😂

Reply:
yea... imagine all the bad jokes we would have to deal with if it were called that...